draft-liu-teas-transport-network-slice-yang.md

```diff
@@ -137,6 +137,22 @@ author:
    The YANG data model in this document conforms to the Network
    Management Datastore Architecture (NMDA) {{!RFC8342}}.
 
+## Use Case Applicability
+
+   In Traffic Engineering (TE)-enabled networks like Layer-0/1 transport (OTN, MW, DWDM), intent topology is useful for routing IETF network slices across varied paths with TE constraints. Thus, most of the use cases for which this model target are transport oriented. Nonetheless, it’s also relevant to non-transport networks like IP/MPLS, where customers may specify topology intents to inform the slice’s realization. These intents help build the logical view of the desired IETF Network Slice service (and its constituent parts), aiding providers in fulfilling slice requests and defining the service instantiation.
+

```

Comment:
A general comment: we need to decide how to call this type of topology: we started with customized topology, now sometimes appear customer topology, intent topology or the opposite , topology intent (as I used in the text for multi-tenant). To avoid confusion for the reader we need to decide one.